ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -34,12 +39,10 @@
       return {
         draggableGroup: Math.random().toString(),
 
-        promptSlots: [
-          { id: 1, text: 'one option' },
-          { id: 2, text: 'two option' },
-          { id: 3, text: 'three option' },
-          { id: 4, text: 'four option' }
-        ],
+        promptSlots: this.localizedInteractiveConfig.elements.map(option => ({
+          id: option.elementId,
+          text: option.text
+        })),
 
         answerSlots: []
```

Comment:
Do these also need to come from `localizedInteractiveConfig`?

Reply:
Yeah. I believe so. As these contain the labels in the right hand column. Great comment!